actions/pacstrap_action.go

```diff
@@ -78,6 +78,16 @@ func NewPacstrapAction() *PacstrapAction {
 	return &d
 }
 
+func (d *PacstrapAction) Verify(context *debos.DebosContext) error {
+	if err := (debos.Command{}.CheckExists("Pacstrap", "pacstrap", "-h")); err != nil {
+		return err
+	}
+	if err := (debos.Command{}.CheckExists("Pacman-key", "pacman-key", "-V")); err != nil {
```

Comment:
Having a helper which takes struct { label string; argv[] string } array should be cleaner IMHO. Not a big deal either way